lib/data/data_sources/remote_data_source/models/user.dart

```diff
@@ -9,5 +10,7 @@ extension UserExtension on User {
         isActive: this.isActive,
         lastSeenAt: this.lastSeenAt,
         metadata: this.metaData,
+        isOnline:
+            this.connectionStatus == UserConnectionStatus.online ? true : false,
```

Comment:
same here, if the comparison is a boolean, why do you need to put true and false as answers?